src/components/Executions/test/useOnlyMyExecutionsFilterState.test.ts

```diff
@@ -1,35 +0,0 @@
-import { loadedFetchable } from 'components/hooks/__mocks__/fetchableData';
-import { useUserProfile } from 'components/hooks/useUserProfile';
-import { useOnlyMyExecutionsFilterState } from 'components/Executions/filters/useOnlyMyExecutionsFilterState';
```

Comment:
Hey, why are we removing this test? The hook itself is not being removed in this PR, we may want to move this under `/filters/test` folder, which doesn't exist yet, and that's probably why I placed it on the parent `/Executions` level

Reply:
Good eyes :) 
This test confused me as the mock to useUserProfile was named mockUseRemoteLiteralMap, and Remote literalMap is not there any more 🙈
Re-named it to mockUseUserProfile and returned test into place.

Thank you for mentioning it :)